doc/about.rst

```diff
@@ -1,10 +1,11 @@
 .. _about:
 
+========
```

Comment:
@Charlie-XIAO The proposed changes with the section style seem to not have any impact on the rendering but I'm wondering if we did not revert back something that we did specifically when switching to the `pydata-sphinx-theme`?

Do we overlook something that might break later in terms of rendering?

Reply:
Nope I don't think anything will be broken. It's essentially lifting each original section one level up to accommodate the new subsections which should be fine.

Reply:
Great so all good.